### PR TITLE
Chapter 3 Application metrics

### PR DESCRIPTION
**Pull request by darienmt**

### CODE REVIEW

File: content/chapter-03/ApplicationMetrics.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+## What to measure on an application?
+
+* Application level metrics ([Resource](https://stackify.com/application-performance-metrics/)):
+  * Error rate, e.g.: Internal error logged per second, 5XX requests/second
+  * Request rate, e.g.: Total requests per second, 2XX requests/second
+  * Response time, e.g.: Average response time, peak response time per second
+  * Availability percentage, e.g.: Percentage of uptime for interval of time
+  * Concurrent users, e.g.: Concurrent user count distribution over a day
+  * User satisfaction using [Apdex score](https://www.apdex.org/)
+  * [Throughput](https://www.cdnetworks.com/en/news/6-critical-web-application-performance-metrics-to-consider/4257). This is very tight to the application at hand. It should be something representing a business value on a unit of time. For example, in an integration application, how many entities were moved from one system to another per using of time.
```
Comment (vkhazin) [inline content/chapter-03/ApplicationMetrics.md:12]: per using of time. =>
per unit of time

File: content/chapter-03/ApplicationMetrics.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  
+  
+* System level metrics ([Resource]())
```
Comment (vkhazin) [inline content/chapter-03/ApplicationMetrics.md:15]: Missing href?

File: content/chapter-03/ApplicationMetrics.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  * CPU utilization, e.g.: CPU utilization percentage per second
```
Comment (vkhazin) [inline content/chapter-03/ApplicationMetrics.md:16]: Is the CPU measured per sec?

File: content/chapter-03/ApplicationMetrics.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  * Memory utilization, e.g.: Peak memory usage, Percentage of memory in used per application
```
Comment (vkhazin) [inline content/chapter-03/ApplicationMetrics.md:17]: Percentage of memory in used per application =>
Percentage of memory in use per application

File: content/chapter-03/ApplicationMetrics.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  * Network utilization, e.g.: Network bandwith usage per second
```
Comment (vkhazin) [inline content/chapter-03/ApplicationMetrics.md:18]: I think there is a more accurate definition of bandwidth utilization than `usage per second`"
https://en.wikibooks.org/wiki/Transwiki:Measuring_network_throughput

**Review by vkhazin — CHANGES_REQUESTED** [review]

File: content/chapter-03/ApplicationMetrics.md (hunk 1 above)
Comment (darienmt) [reply content/chapter-03/ApplicationMetrics.md:12]: Done

File: content/chapter-03/ApplicationMetrics.md (hunk 2 above)
Comment (darienmt) [reply content/chapter-03/ApplicationMetrics.md:15]: Done

File: content/chapter-03/ApplicationMetrics.md (hunk 3 above)
Comment (darienmt) [reply content/chapter-03/ApplicationMetrics.md:16]: The idea I want to put is that the measurement is always per time unit unless you just sampling it. It is not common though. Usually is sampled so I will remove that.

File: content/chapter-03/ApplicationMetrics.md (hunk 4 above)
Comment (darienmt) [reply content/chapter-03/ApplicationMetrics.md:17]: Done

**Pull request merged**